libmozdata/hgmozilla.py

```diff
@@ -241,3 +246,41 @@ def get(paths, channel='nightly', node='tip'):
             FileInfo(queries=queries).wait()
 
         return data
+
+
+class HGMozilla(object):
```

Comment:
If we had the same interface, when we want to add new things, we could add them to both classes. This way the user of the library doesn't have to worry about which one they're using (the faster one or the slower one).